src/Public/js/nipsweb.managelibrary.js

```diff
@@ -122,34 +138,35 @@ var Library = function () {
                     var track_album = "";
                     var track_position = "";
 
-                    // Build a list of tracks from the lastfm responses and store it in a drop
-                    // down list
-                    var select = "";
-                    if (!manual_track) {
-                        select = $('<select></select>')
-                        .attr('name', response.fileid).attr('id', 'centralupload-' + i);
-                        $.each(
-                            response.analysis,
-                            function (
-                                key,
-                                value
-                            ) {
-                                select.append('<option value="' + value.title + ':-:' + value.artist + '">' + value.title + ' by ' + value.artist + '</option>');
+
+                    if (manual_track) {
+                        if (response.analysis) {
+                            //Otherwise, if we got an analysis from the ID3 tags, prefill the textboxes.
+                            function decodeHtml(html) {
```

Comment:
Erm, well this is apparently the best method I could find on Google for doing it this way round? Converting & to &amp; etc is easy, but apparently the other way is hard...